java/java.lsp.server/vscode/src/explorer.ts

```diff
@@ -92,11 +97,11 @@ class VisualizerProvider extends vscode.Disposable implements CustomizableTreeDa
   private root: Visualizer;
   private treeData : Map<number, Visualizer> = new Map();
   private decorators : TreeItemDecorator<Visualizer>[] = [];
-  private pendingRefresh : Set<number> = new Set();
-
+  
   constructor(
     private client: LanguageClient,
     private ts : TreeViewService,
+    private log : vscode.OutputChannel,
```

Comment:
`this.log = log` is missing in the constructor body.

Reply:
I thought I've used [Parameter properties](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties) in the constructor with an automatic assignment to the member.

Reply:
Aha, O.K. then it's fine. Thanks for explanation.